browserify.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var webkitAssign = require('./index');
+
+module.exports = function (file) {
+	return webkitAssign();
+};
```

Comment:
That doesn't work because webkitAssign() in index.js checks whether the first parameter is a string.  Since Browserify passes the filename as the first param, it tries to call transformCode(code) which fails because Browserify expects a stream response.

I hoped it would work without any wrapper, but I tested it and Browserify wouldn't work as-is.  I considered modifying index.js but it ended up too convoluted.